src/main/resources/themes/vscode_dark.xml

```diff
@@ -523,6 +523,11 @@
                 <option name="FOREGROUND" value="569cd6"/>
             </value>
         </option>
+        <option name="HTML_CUSTOM_TAG_NAME">
+            <value>
+                <option name="FOREGROUND" value="40c8ae"/>
```

Comment:
To keep it consistent with rest of the changes please update this to: 

```suggestion
                <option name="FOREGROUND" value="4ec9b0"/>
```

Reply:
Done ✔️